<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>675</width>
    <height>546</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QGraphicsView" name="inputImageView">
        <property name="interactive">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QGraphicsView" name="referenceImageView">
        <property name="interactive">
         <bool>false</bool>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="0" column="1" colspan="2">
     <widget class="QGraphicsView" name="bestImageView">
      <property name="interactive">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QPushButton" name="inputImageButton">
        <property name="text">
         <string>Wczytaj obraz wejsciowy</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="referenceImageButton">
        <property name="text">
         <string>Wczytaj obraz referencyjny</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="startButton">
        <property name="text">
         <string>Start</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="stopButton">
        <property name="text">
         <string>Stop</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="optionsButton">
        <property name="text">
         <string>Opcje</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="1" column="1">
     <widget class="QGroupBox" name="infoBox">
      <property name="title">
       <string>Informacje</string>
      </property>
      <layout class="QFormLayout" name="formLayout">
       <item row="0" column="0">
        <widget class="QLabel" name="generationNumberLabel">
         <property name="text">
          <string>Number generacji</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QLCDNumber" name="generationNumber">
         <property name="numDigits">
          <number>5</number>
         </property>
         <property name="digitCount">
          <number>5</number>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="operationLabel">
         <property name="text">
          <string>Wykonywana operacja</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QLCDNumber" name="individualNumber"/>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="individualNumberLabel">
         <property name="text">
          <string>Number osobnika</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QLabel" name="operationNameLabel">
         <property name="text">
          <string>TextLabel</string>
         </property>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="maxGenerationNumberLabel">
         <property name="text">
          <string>Liczba generacji</string>
         </property>
        </widget>
       </item>
       <item row="4" column="0">
        <widget class="QLabel" name="ideal">
         <property name="text">
          <string>Minimalny wynik</string>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <widget class="QLCDNumber" name="lcdNumber"/>
       </item>
       <item row="4" column="1">
        <widget class="QLCDNumber" name="lcdNumber_2"/>
       </item>
      </layout>
      <zorder>inputImageView</zorder>
      <zorder>generationNumberLabel</zorder>
      <zorder>operationLabel</zorder>
      <zorder>generationNumber</zorder>
      <zorder>individualNumber</zorder>
      <zorder>individualNumberLabel</zorder>
      <zorder>operationNameLabel</zorder>
      <zorder>maxGenerationNumberLabel</zorder>
      <zorder>ideal</zorder>
      <zorder>lcdNumber</zorder>
      <zorder>lcdNumber_2</zorder>
     </widget>
    </item>
    <item row="1" column="2">
     <widget class="QGroupBox" name="bestImageInfoBox">
      <property name="title">
       <string>Najlepsze rozwiazanie</string>
      </property>
      <layout class="QFormLayout" name="formLayout_3">
       <property name="fieldGrowthPolicy">
        <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
       </property>
       <item row="0" column="0">
        <widget class="QLabel" name="resultLabelBest">
         <property name="text">
          <string>Wynik</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="generationNumberLabelBest">
         <property name="text">
          <string>Numer generacji</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QLCDNumber" name="generationNumberBest"/>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="individualNumberLabelBest">
         <property name="text">
          <string>Numer osobnika</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QLCDNumber" name="individualNumberBest"/>
       </item>
       <item row="0" column="1">
        <widget class="QLCDNumber" name="resultNumberBest"/>
       </item>
      </layout>
      <zorder>infoBox</zorder>
      <zorder>infoBox</zorder>
      <zorder>generationNumberLabelBest</zorder>
      <zorder>individualNumberLabelBest</zorder>
      <zorder>generationNumberBest</zorder>
      <zorder>individualNumberBest</zorder>
      <zorder>resultLabelBest</zorder>
      <zorder>resultNumberBest</zorder>
     </widget>
    </item>
   </layout>
   <zorder>bestImageView</zorder>
   <zorder>verticalLayoutWidget_2</zorder>
   <zorder>verticalLayoutWidget</zorder>
   <zorder>infoBox</zorder>
   <zorder>inputImageView</zorder>
   <zorder>bestImageInfoBox</zorder>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
